### PR TITLE
fix code sample in "Use Promises, not callbacks" section

### PR DESCRIPTION
**Pull request by ateshuseyin**
I fixed package names in good code sample in "Use Promises, not callbacks" section.

### CODE REVIEW

**Comment by ryanmcdermott** [comment]
Thanks for looking into this! Apparently `fs-promise` is deprecated? https://github.com/kevinbeaty/fs-promise

File: README.md [hunk 1]
```diff
@@ -1773,8 +1773,8 @@ get('https://en.wikipedia.org/wiki/Robert_Cecil_Martin', (requestErr, response)
 
 **Good:**
 ```javascript
-import { get } from 'request';
-import { writeFile } from 'fs';
+import { get } from 'request-promise';
+import { writeFile } from 'fs-promise';
```
Comment (felquis) [inline README.md]: Use `fs-extra` instead!

**Comment by styfle** [comment]
Node 8 actually has a native solution for wrapping core APIs in promises.
See this [blog post](https://medium.com/@styfle/promises-in-node-js-8-x-core-d6a8a93e85a2) for more info.

**Comment by felquis** [comment]
I don't see advantage using the built in promisify than using fs-extra.

**Comment by styfle** [comment]
> I don't see advantage using the built in promisify than using fs-extra.

The advantage is one less dependency.

**Comment by ryanmcdermott** [comment]
Closing this as `request` has been deprecated. I will fix these sections at a later date and likely use `fetch`.

**Pull request closed without merging**